packages/ui-text-area/src/TextArea/index.js

```diff
@@ -150,10 +153,12 @@ class TextArea extends Component {
 
   componentDidMount() {
     this.autoGrow()
+    this.props.makeStyles()
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
`prevProps, prevState, snapshot ` Are these necessary?

Reply:
WebStorm gives a warning when these are missing, and it might cause a build error when we convert this to TypeScript. But for now it doesnt change anything